Common subdirectories: vlan.orig/CVS and vlan/CVS
diff -u vlan.orig/MakeInclude vlan/MakeInclude
--- vlan.orig/MakeInclude	2002-02-09 07:46:18.000000000 +0100
+++ vlan/MakeInclude	2002-06-16 17:25:01.000000000 +0200
@@ -16,7 +16,7 @@
 endif
 
 ## You may need to change this linux/include part.
-CCFLAGS = -g -D_GNU_SOURCE -Wall -I${HOME}/linux/include
+CCFLAGS = $(RPM_OPT_FLAGS) -D_GNU_SOURCE -Wall -I$/usr/include/linux
 LDLIBS = # -lm #-lnsl # -lsocket
 
 ARM_TC_BIN = ${HOME}/Intrinsyc/bin
Common subdirectories: vlan.orig/contrib and vlan/contrib
